e2e_playwright/st_dataframe_interactions.py

```diff
@@ -40,3 +40,35 @@
 
 
 st.data_editor(random_df, num_rows="dynamic")
+
+
+cell_overlay_test_df = pd.DataFrame(
+    {
+        "big_numbers": [1231231.41, 12012],
+        "text": ["hello\nworld", "foo"],
+    }
+)
+
+cell_overlay_test_column_config = {
+    # the e2e test requires all cells to medium width
```

Comment:
nit: could you add a short comment _why_ the e2e test requires this, please?

Reply:
Added 👍